uint16 nb_set_points
uint16[] set_points_x
---
float32[4] motor_vel
